pyproject.toml

```diff
@@ -8,36 +8,37 @@ authors = ["Leon Handreke <leonh@ndreke.de>"]
 python = "^3.10"
 Django = "~3.2.13"
```

Comment:
Besides 3.2 is LTS and I can't see other reasons except bugfixes, but is there a reason we don't update to 4.x Django?

Reply:
as far as I remember, when also trying Django 4, django-ldap is incompatible with Django 4 and it doesn't seem very actively maintained.
We're looking into replacing LDAP with Keycloak at WirMarkt. When that's successful, we'll come back to you with our experience.